src/supermarket/config/routes.rb

```diff
@@ -73,48 +73,13 @@
     get 'versions/:version' => 'cookbook_versions#show', as: :version, constraints: { version: VERSION_PATTERN }
   end
 
-  resources :icla_signatures, path: 'icla-signatures', constraints: proc { ROLLOUT.active?(:cla) && ROLLOUT.active?(:github) } do
-    collection do
-      post :re_sign, path: 're-sign'
-      get :agreement
-    end
-  end
-
   resources :collaborators, only: [:index, :new, :create, :destroy] do
     member do
       put :transfer
       delete :destroy_group
     end
   end
 
-  resources :ccla_signatures, path: 'ccla-signatures', constraints: proc { ROLLOUT.active?(:cla) && ROLLOUT.active?(:github) } do
-    collection do
-      post :re_sign, path: 're-sign'
-      get :agreement
-    end
-
-    member do
-      get :contributors
-    end
-
-    resources :contributor_requests, only: [:create], constraints: proc { ROLLOUT.active?(:join_ccla) && ROLLOUT.active?(:github) } do
-      member do
-        get :accept
-        get :decline
-      end
-    end
-  end
-
-  namespace :curry, constraints: proc { ROLLOUT.active?(:cla) && ROLLOUT.active?(:github) } do
-    resources :repositories, only: [:index, :create, :destroy] do
-      member do
-        post :evaluate
-      end
-    end
-
-    resources :pull_request_updates, only: [:create]
```

Comment:
I could maybe see keeping this route—the former GitHub webhook handler—to return a 4xx something "Nothing to be found here anymore" error.